components/k8s-model-server/images/Dockerfile.cpu

```diff
@@ -12,42 +12,75 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:16.04
+FROM centos:7
 
-MAINTAINER Kenneth Owens <kowens@google.com>
+MAINTAINER Chad Roberts <croberts@redhat.com>
 
 ENV MS_USER=model-server
+USER root
 
-RUN     apt-get update && apt-get install -y software-properties-common && \
-        # TODO (pmackinn) see if explicit libstdc++6 is still really required with next pin
-        add-apt-repository ppa:ubuntu-toolchain-r/test -y && \
-        apt-get update && apt-get install -y \
+RUN yum --enablerepo=extras install -y epel-release && \
+    yum update -y && yum install -y \
         build-essential \
         curl \
-        libcurl3-dev \
         git \
-        libfreetype6-dev \
-        libpng12-dev \
-        libstdc++6 \
-        libzmq3-dev \
-        pkg-config \
-        python-dev \
-        python-numpy \
+        gcc \
+        gcc-c++ \
+        mlocate \
+        patch \
+        python-devel \
         python-pip \
         swig \
         zip \
-        zlib1g-dev && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+        wget \
+        unzip \
+        which \
+        && \
+    yum clean all
 
-RUN curl -O http://storage.googleapis.com/tensorflow-serving-apt/pool/tensorflow-model-server/t/tensorflow-model-server/tensorflow-model-server_1.6.0_all.deb
-RUN dpkg -i tensorflow-model-server_1.6.0_all.deb
+# Set up grpc
+ENV LD_LIBRARY_PATH=/opt/rh/python27/root/usr/lib64
+RUN pip install --upgrade pip
+RUN pip install mock grpcio numpy
 
+# Set up Bazel.
+
+ENV BAZELRC /root/.bazelrc
```

Comment:
@lluunn I'm guessing the switch to centos forces us to build TF serving ourself since they probably don't provide a YUM package.

I think we're already building it in the case of the GPU image. If that's the case I think being consistent is good. Does TF-serving publish a deb for GPU?